packages/web-components/src/components.d.ts

```diff
@@ -265,9 +265,9 @@ export namespace Components {
          */
         "secondaryLabel"?: string;
         /**
-          * If `true`, the primary button will submit form data when clicked.
+          * Having this attribut present will set the type of this button as 'submit'. A value of: `prevent` --will  triger the onsubmit callback on form, but won't submit the form; `skip` --will submit the form but not trigger onsubmit callback;
```

Comment:
Typo: "attribut" should be "attribute".
Type: "--will" can just be "will"
Typo: "triger" should be "trigger"
Type: "callback on form" should be "callback on the form"
Typo: "trigger onsubmit callback;" shoudl be "trigger the onsubmit callback."